src/sandstorm/web-session.capnp

```diff
@@ -140,6 +145,9 @@ interface WebSession @0xa50711a14d35a8ce extends(Grain.UiSession) {
     struct Header {
       name @0 :Text;  # lower-cased name
       value @1 :Text;
+      prefixed @2 :Bool = false;
```

Comment:
Mixing non-prefixed and prefixed headers like this worries me a bit -- it feels like a pattern that could lead to injections when receiving code doesn't carefully check this boolean.

Would it make sense instead to separate the whitelisted headers and prefixed headers into two lists? Or maybe we should simply not strip the prefix from the name in Header.name, to avoid ambiguity?